BeeTracker/app/src/main/res/layout/activity_learn.xml

```diff
@@ -4,11 +4,15 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="#F6F6F6"
     tools:context="com.blueprint.foe.beetracker.LearnActivity">
-
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="learn activity" />
-
+    <ListView
+        android:id="@+id/factsListView"
+        android:animateLayoutChanges="true"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_alignParentTop="true"
+        android:layout_alignParentStart="true"
+        android:divider="@null"
+        android:dividerHeight="0dp"/>
```

Comment:
We should be putting our dimensions in the dimen.xml file. I'm not great at doing this, so don't follow my lead on that :P but we should all be putting them there

Reply:
0dp is fine